web-client/src/components/CartItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/ui/button';
+import { Card, CardHeader, CardContent, CardFooter } from '@/components/ui/card';
+import { Trash2 } from "lucide-react";
+
+interface CartItemProps {
+    sockId: number;
+    sockVariantId: number;
+    name: string;
+    size: string;
+    price: number;
+    quantity: number;
+    imageUrl: string;
+    onIncrease: () => void;
+    onDecrease: () => void;
+    onRemove: () => void;
+}
+
+export default function CartItem({
+    name,
+    size,
+    price,
+    quantity,
+    imageUrl,
+    onIncrease,
+    onDecrease,
+    onRemove
+}: CartItemProps) {
+    return (
+        <Card className="w-full mx-auto mb-4" >
+            <CardContent className="flex flex-col gap-4 md:flex-row justify-between p-4">
+                <div className="flex gap-4 items-center">
+                    <img src={imageUrl} alt={name} className="w-24 h-24 object-cover rounded-md" />
+                    <div className="text-lg font-semibold">
+                        <h3 className="mb-1">{name}</h3>
+                        <p className="text-sm mb-1 text-gray-400">Size: {size}</p>
```

Comment:
Add the "strong" tag to make "Size:" bold

```jsx
<p className="mb-1 text-sm text-gray-400">
   <strong>Size:</strong> {item.size}
</p>
```

Reply:
Fixed. I had to do {size} instead of {item.size}. 
item.size did not work.